docs/pages/upgrading/automatic-agent-updates.mdx

```diff
@@ -0,0 +1,242 @@
+---
+title: Set up Automatic Agent Updates
+description: Describes how to set up automatic agent updates.
+---
+
+We strongly recommend that Teleport users set up automatic agent updates to
+ensure that the version of Teleport running on agents remains compatible with
+the version running on the Auth Service and Proxy Service. If an agent does not
+maintain [version compatibility](./overview.mdx) with your Teleport cluster,
+connections to those agents will become degraded or lost.
+
+Cloud-hosted Teleport clusters are updated on a weekly basis. Major version
+updates are performed every 4 months. You can monitor and subscribe to the
+[Teleport Status](https://status.teleport.sh/) page to be notified of scheduled
+updates.
+
+Teleport supports automatic agent updates for systemd-based Linux distributions
+using `apt`, `yum`, and `zypper` package managers, as well as Kubernetes
+clusters. 
+
+This guide explains how to enable automatic updates for Teleport agents on
+cloud-hosted Teleport Enterprise clusters. 
+
+## How it works
+
+When automatic updates are enabled, a Teleport updater is installed alongside
+each Teleport agent. The updater communicates with the Teleport Proxy Service to
+determine when an update is available. When an update is available, the updater
+will update the Teleport agent during the next maintenance window. However, if a
+critical update is available, the Teleport agent will be updated outside the
+regular maintenance window.
+
+## Prerequisites
+
+- A cloud-hosted Teleport Enterprise cluster. If you do not have one, [sign
+  up](https://goteleport.com/signup) for a free trial or consult the [Update
+  Reference](./reference.mdx) to read about manual updates or automatic agent
+  updates for self-hosted clusters.
+
+- Familiarity with the [Upgrading Compatibility Overview](./overview.mdx) guide,
+  which describes the sequence in which to upgrade components in your cluster.
+
+- (!docs/pages/includes/cloud/tctl-tsh-prerequisite.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/3. Find agents to enroll in automatic updates
+
+Use the `tctl inventory ls` command to list connected agents along with their current
+version. Use the `--upgrader=none` flag to list agents that are not enrolled in automatic
+updates.
+
+
+```code
+$ tctl inventory ls --upgrader=none
+Server ID                            Hostname      Services Version Upgrader
+------------------------------------ ------------- -------- ------- --------
+00000000-0000-0000-0000-000000000000 ip-10-1-6-130 Node     v14.4.5 none
+...
+```
+
+Note that the `inventory ls` command will also list `teleport-auth` and
+`teleport-proxy` services. These services are managed by the Teleport team, and
+updates will be performed automatically.
+
+## Step 2/3. Enroll agents on Linux servers in automatic updates
```

Comment:
It seems like this guide is targeting clusters with existing agents that aren't yet enrolled in auto-upgrades? We should make this clear I think. Cloud users enrolling new agents should get automatic upgrades enabled by default, is that correct?

Reply:
Edited the prerequisites. Step 2 shows the user how to identify agents that are not enrolled in automatic updates.